client/src/pages/OnBoard.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from 'react';
+//Material-ui imports
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Background from '../elements/Background';
+import IconButton from '@material-ui/core/IconButton';
+import RemoveCircleIcon from '@material-ui/icons/RemoveCircle';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(12),
+    padding: theme.spacing(5),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  headerText: {
+    marginTop: theme.spacing(10)
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    textAlign: 'center'
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    color: '#fff',
+    textTransform: 'none',
+    fontSize: 15
+  },
+  add: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    textTransform: 'none',
+    fontSize: 16
+  },
+  removeIcon: {
+    color: theme.palette.button.remove
+  }
```

Comment:
A lot of these styles are identical to the previous two pages. Can we create a theme specific to the onboarding and sign in flow that provides these to keep it DRY?

Reply:
We can either add this to the material-ui theme and override styles or create components that extend a component with certain styles added to it if we want more customization. Something like:
```

const StyledPaper = ({children, ...otherProps}) => {
      const classes = useStyles(); 
      return <Paper className={classes.paper} {...otherProps}>{children}</Paper>
}
```

Reply:
@skassam21 I've already done that with Background ... but thx for suggestion, will rewrite for repeating components